### DISCUSSION TITLE
编译发生错误

### DISCUSSION THREAD

**Discussion by De4tsh** [tangly1024/NotionNext#178] [Q&A] [answered]
![QQ截图20220414171709](https://user-images.githubusercontent.com/52061639/163354598-6e48f6fe-f3c6-4eef-a33f-a1bd1eac1f04.png)
您好请问，这个错误如何排除

**Comment by tangly1024** [comment]
🤔看起来像是谋篇文章未设置`分类`导致的错误，检查一下文章列表。

同时我也会提交一个补丁修复这个错误。

**Reply by tangly1024** [reply]
最新的[代码](https://github.com/tangly1024/NotionNext/pull/179)中修复了此问题，请尝试一下更新

**Comment by De4tsh** [comment, marked as answer]
非常感谢，已经解决。
问题原因：Notion中未将category字段的种类选为Select

**Reply by vinson-lee01** [reply]
Expect support

**Comment by vinson-lee01** [comment]
> ![QQ截图20220414171709](https://user-images.githubusercontent.com/52061639/163354598-6e48f6fe-f3c6-4eef-a33f-a1bd1eac1f04.png) 您好请问，这个错误如何排除

Expect support
